compiler/src/iree/compiler/Dialect/Stream/Transforms/test/encode_host_tensors_packing_i1_attr.mlir

```diff
@@ -0,0 +1,22 @@
+// RUN: iree-opt --split-input-file --iree-stream-encode-host-tensors %s | FileCheck %s
```

Comment:
since this uses the same flags as the main test you can move it back into it (encode_host_tensors.mlir) - each individual .mlir file adds test overhead and makes it harder to manage the tests